service/matching/version_sets.go

```diff
@@ -39,12 +39,8 @@ import (
 )
 
 var (
-	// TODO: all of these errors are temporary, we'll handle all these cases in future PRs
-	errBuildIDNotFound                   = serviceerror.NewInvalidArgument("build ID not found")
-	errEmptyVersioningData               = serviceerror.NewInvalidArgument("versioning data is empty")
-	errPollWithVersionOnUnversionedQueue = serviceerror.NewInvalidArgument("poll with version capabilities on unversioned queue")
-	errPollOnVersionedQueueWithNoVersion = serviceerror.NewInvalidArgument("poll on versioned queue with no version capabilities")
-	errVersionedTaskForUnversionedQueue  = serviceerror.NewInvalidArgument("got task with version stamp for unversioned queue")
+	// This shouldn't happen, if we have versioning data we should have at least one set.
```

Comment:
this is the one remaining error here, and it shouldn't happen if there are no bugs in the mutation stuff

Reply:
Awesome